index.html

```diff
@@ -4310,33 +4310,29 @@ <h2>
 
     <section>
         <h2>
-Persistence of DIDs
+DIDs as URNs
         </h2>
 
         <p>
-Although a <a>DID</a> is able to function as a Uniform Resource Name (URN)—a
-"persistent, location-independent resource identifier"as defined by [[RFC8141]]—
-this is not a requirement of all DIDs. Rather the persistence of a <a>DID</a>,
-is determined by the <a>DID controller</a>.
-
-If a <a>DID controller</a> intends for a <a>DID</a> to function as a URN, there
-are two important security considerations:
+If desired by a <a>DID controller</a>, a <a>DID</a> is capable of fulfilling
+the functions of a Uniform Resource Name (URN) as defined by [[RFC8141]], i.e.,
+"a persistent, location-independent resource identifier". A <a>DID controller</a>
+who intends to use a <a>DID</a> for this purpose is advised to follow the
+security considerations in [[RFC8141]]. In particular:
         </p>
         <ol start="1">
           <li>
-The <a>DID controller</a> should choose a <a>DID method</a> that supports their
-requirements for persistence. <a>DID controllers</a> are advised to refer to the
-Decentralized Characteristics Rubric [[?DID-RUBRIC]] to decide which
-<a>DID method</a> best meets their persistence requirements.
+The <a>DID controller</a> should choose a <a>DID method</a> that supports the
+<a>DID controller</a>'s requirements for persistence. It is recommended to
```

Comment:
This *may* require a tweak to the terminology doc.  I know ReSpec takes care of (most) plurals automatically; I'm not sure about possessives.  Definitely, `</a>'s` will have an ugly gap preceding the apostrophe, so that should not be kept.  Might be that `the <a>DID controller</a>'s` should instead be changed to `their`.
```suggestion
<a>DID controller's</a> requirements for persistence. It is recommended to
```